src/pagingscrollview/src/NIPagingScrollView.m

```diff
@@ -393,6 +421,11 @@ - (void)setFrame:(CGRect)frame {
   [self layoutVisiblePages];
 }
 
+- (void)layoutSubviews {
+  _scrollView.contentSize = [self contentSizeForPagingScrollView];
+  [self layoutVisiblePages];
```

Comment:
I thought we did, because the combo of setting the content size and invoking layoutVisiblePages is definitely required.  However, I can move that to setPageInset and that's a lot simpler.